src/test/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcherSpec.scala

```diff
@@ -256,14 +256,24 @@ class BlockFetcherSpec
       val checkpointBlock = (new CheckpointBlockGenerator)
         .generate(
           firstBlocksBatch.last,
-          Checkpoint(CheckpointingTestHelpers.createCheckpointSignatures(Seq(crypto.generateKeyPair(secureRandom)), firstBlocksBatch.last.hash))
+          Checkpoint(
```

Comment:
If there are not changes in this file it would be good to revert changes, for better review. The PR is pretty complex.

Reply:
It's a change introduced by scalaFmt...